examples/consumer/main.go

```diff
@@ -71,6 +72,12 @@ func main() {
 	flag.Parse()
 
 	nodeName := os.Getenv("NODE_NAME")
```

Comment:
the consumer doesn't have to do sanitize, the node name. Because  node  replacement for URI  is done by the cloud-event-proxy framework